scripts/popup.js

```diff
@@ -315,6 +315,32 @@ function borrow_books(){
     });
 }
 
+function show_news(){
+    chrome.tabs.query({active: true,currentWindow:true},function(tabs){
+        url=tabs[0].url;
+        tabId=tabs[0].id;
+        chrome.storage.sync.get(['news'],function(event){
+            if(event.news==true){
+                if(url.includes("https://www.huffingtonpost"||"https://www.nytimes"||"https://www.forbes"||"https://www.washingtonpost")){
```

Comment:
Why do we use this only for these sites?
I tried it with https://edition.cnn.com/2018/09/01/politics/trump-administration-summer-hell-drama-key-moments/index.html and it worked fine.

Reply:
I was told by Max that to use this feature for these sites only.